Sources/Public/ItemViews/SwiftUIWrapperView.swift

```diff
@@ -47,16 +47,20 @@ public final class SwiftUIWrapperView<Content: View>: UIView {
 
   // MARK: Public
 
+  public override class var layerClass: AnyClass {
+    CATransformLayer.self
+  }
+
   public override var isAccessibilityElement: Bool {
     get { false }
     set { }
   }
 
-  public override func didMoveToWindow() {
-    super.didMoveToWindow()
-
-    if window != nil {
-      setUpHostingControllerIfNeeded()
+  public override var isHidden: Bool {
+    didSet {
+      if isHidden {
+        hostingController.rootView = AnyView(EmptyView())
```

Comment:
not ideal, but way faster than the previous strategy of giving every view an ID

Reply:
We could also add an API in the future to opt-out of this behavior if someone doesn't need `onAppear` or `onDisappear` callbacks in their views, which would get them back ~20% CPU usage when scrolling fast 🤯